core/server/data/migrations/versions/4.0/20-regenerate-posts-html.js

```diff
@@ -0,0 +1,75 @@
+const logging = require('../../../../../shared/logging');
+const {createIrreversibleMigration} = require('../../utils');
+const mobiledocLib = require('../../../../lib/mobiledoc');
+const htmlToText = require('html-to-text');
+
+module.exports = createIrreversibleMigration(async (knex) => {
+    logging.info('Starting re-generation of posts html.');
+
+    await knex.transaction(async (trx) => {
+        // get list of posts ids, use .forUpdate to lock rows until the transaction is finished
+        const postIdRows = await knex('posts')
+            .transacting(trx)
+            .forUpdate()
+            .select('id');
```

Comment:
You could exclude the ids of post without mobiledoc here, but I guess it's not really worth the trouble because it's a micro-optimization.

Reply:
I left it in because the logs for invalid/missing mobiledoc could be useful, missing mobiledoc means an invalid post record that could be indicative of other issues.